components/image/image.css

```diff
@@ -0,0 +1,17 @@
+.reponsive-image {
+    display: flex;
+}
+
+.round-image{
+    width: 20rem;
+    height: 20rem;
+    border-radius: 50%;
```

Comment:
Suggestion: If you want to make it responsive you can use max-width and max-height properties